<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>АРМ Инспектора</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Автоматизированное рабочее место инспектора&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableView {
         Color: white; /* text color in the table */
         Gridline-color: black; /*The inner frame color*/
         Background-color: rgb(108, 108, 108); /* background color in the table*/
    alternate-background-color: rgb(64, 64, 64);
         Selection-color: white; /* The text color of the selected area*/
         Selection-background-color: rgb(77, 77, 77); /*The background color of the selected area*/
    border: 2px groove gray;
    border-radius: 0px;
    padding: 2px 4px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>АРМ Администратора</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionListUsers"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>АРМ Юриста</string>
    </property>
    <addaction name="actionReport"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAccount">
   <property name="text">
    <string>Account</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Отчёт об АП</string>
   </property>
  </action>
  <action name="actionListUsers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../xmlParser/images/information.png</normaloff>../../xmlParser/images/information.png</iconset>
   </property>
   <property name="text">
    <string>Список пользователей</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionReport">
   <property name="text">
    <string>Отчёт</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
